### PR TITLE
add gatsby-contentful-auth0 sample app

### PR DESCRIPTION
**Pull request by praveenweb**
### Description

This is a sample app with Gatsby, Contentful and Auth0, demonstrating how to build a music playlist app with Hasura Remote Joins.

### Affected components 

- Community Content

### CODE REVIEW

File: community/sample-apps/gatsby-contentful-auth0/app/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (shahidhk) [inline community/sample-apps/gatsby-contentful-auth0/app/LICENSE]: @praveenweb Is it okay if we remove this file? We already have a top-level license.

File: community/sample-apps/gatsby-contentful-auth0/app/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# gatsby-contentful-auth0
```
Comment (shahidhk) [inline community/sample-apps/gatsby-contentful-auth0/app/README.md]: Is this readme same as the top-level one?

Comment (praveenweb) [reply community/sample-apps/gatsby-contentful-auth0/app/README.md]: The top level is more detailed with setup instructions for contentful, auth0 etc. This is more specific to running the gatsby app alone.

File: community/sample-apps/gatsby-contentful-auth0/app/LICENSE (hunk 1 above)
Comment (praveenweb) [reply community/sample-apps/gatsby-contentful-auth0/app/LICENSE]: Sure, can be removed.

**Pull request merged**